isis/src/qisis/objs/CnetEditorView/CnetEditorView.h

```diff
@@ -79,6 +81,9 @@ class CnetEditorView : public AbstractProjectItemView {
     void load(XmlStackedHandlerReader *xmlReader);
     void save(QXmlStreamWriter &stream, Project *project, FileName newProjectRoot) const;
 
+    private:
```

Comment:
You have two private sections. You should combine them.

Reply:
This is done throughout Isis:  private methods in one private section vs member variables in another private section.  I can't find anything in coding standards regarding this.  Has this changed?